docs/greptimecloud/integrations/otlp.md

```diff
@@ -0,0 +1,10 @@
+# OpenTelemetry Protocol(OTLP)
+
+GreptimeDB can be used as a collector to receive OpenTelemetry Metrics via the OTLP/HTTP protocol. To send OpenTelemetry Metrics to GreptimeDB through Opentelemetry exporter SDK libraries, use the following information:
+
+* URL: `https://<host>/v1/otlp/v1/metrics`
+* Method: `POST`
+* Headers:
+  * `Content-Type`: `application/x-protobuf`
```

Comment:
No need to specify content-type and method here. OpenTelemetry SDK should take care of the information.

Reply:
* The `Method` can be removed
* The `Content-type` should be written because developers need to use the SDK which supports `application/x-protobuf`, and each languages has it's own way to do that. Maybe I can write more information about `Content-type`.